dali/operators/image/distortion/jpeg_compression_distortion_op_cpu.cc

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <vector>
+#include <opencv2/opencv.hpp>
+#include "dali/operators/image/distortion/jpeg_compression_distortion_op.h"
+
+namespace dali {
+
+DALI_SCHEMA(JpegCompressionDistortion)
+    .DocStr(R"code(Produces JPEG-like distortion in RGB images.
+
+The level of degradation of the image can be controlled with the ``quality`` argument,
+)code")
```

Comment:
IMHO, there should be a clear explanation, what is a "JPEG-like distortion"

Reply:
```suggestion
    .DocStr(R"code(Introduces JPEG compression artifacts to RGB images.

JPEG is a lossy compression format which exploits characteristics of natural images and human visual system to achieve high compression ratios. The information loss originates from sampling the color information at a lower spatial resolution than the brightness and from representing high frequency components of the image with a lower effective bit depth. The conversion to frequency domain and quantization is applied independently to 8x8 pixel blocks, which introduces discontinuities known as block-artifacts.

This operation produces images by subjecting the input to a transformation that mimics JPEG compression with given ``quality`` factor followed by decompression .
)code")
```
?

Reply:
done